### DISCUSSION TITLE
How to use match import inside the Fastfile?

### DISCUSSION THREAD

**Discussion by miscampbell** [fastlane/fastlane#17418] [Q&A]
Hi

I would like to use match import within my fastfile but I am struggling to understand how to do this, below are examples of what I have tried so far.

Any pointers would be great to understand if this possible or do I have to use terminal to use match import?

```
lane :match_app do
    cer_file_path = 'test'
    p12_file_path = 'test'
    provision_profile_file_path = 'test'    

    params = {
      git_url: "git_url",
      shallow_clone: true,
      git_branch: "testapp",
      clone_branch_directly: true,
      type: "development",
      readonly: false,
      username: "test@test.com",
      team_id: "test"
    }

    Importer.new.import_cert(
      params,
      cert_path: cer_file_path,
      p12_path: p12_file_path,
      profile_path: provision_profile_file_path,
    )
  end
```

**Comment by rogerluan** [comment]
Hey @miscampbell 👋 

What's the issue with the code above? You might need to add `Match::` before `Importer`, but other than that, looks like that code would work.
Alternatively, I _believe_ you could call simply `import_cert(…)` and it'd work, as it's available as an action.

**Reply by tobiv** [reply]
I don't know if this is still active/relevant, but I just ran into the same issue. I tried with Match::Importer.new.import_cert(). This is the error I get:

    Fastfile:30:in `block (2 levels) in parsing_binding': [!] uninitialized constant Fastlane::FastFile::Match (NameError)
